src/Discord.Net.Commands/Attributes/PreconditionAttribute.cs

```diff
@@ -6,6 +6,13 @@ namespace Discord.Commands
     [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = true, Inherited = true)]
     public abstract class PreconditionAttribute : Attribute
     {
+        /// <summary>
+        /// Specify a group that this precondition belongs to. Preconditions of the same group require only one
+        /// of the preconditions to pass in order to be successful (A || B). Specifying <see cref="Group"/> = 0
+        /// or not at all will require *all* preconditions to pass, just like normal (A && B).
```

Comment:
Minor nitpick: I'm not sure how `&&` will work with docfx considering this is badly formed XML; I'd suggest replacing with `&amp;&amp;` to ensure that it works best.